src/ondisk/arrows.rs

```diff
@@ -83,8 +83,8 @@ where
             ProjectionMask::roots(schema_descriptor, [2]),
             move |record_batch| {
                 upper_cmp(
-                    record_batch.column(0),
                     &upper_key.to_arrow_datum() as &dyn Datum,
+                    record_batch.column(0),
```

Comment:
is this a mistake?

Reply:
`Bound::Included` maps to `cmp::gt_eq` and `Bound::Excluded` maps to `cmp::gt`. So, It should be like `upper_key > record` and `record > lower_key`, that is the position should exchanged